ios/src/ViewController.mm

```diff
@@ -201,16 +200,18 @@ - (void)setupGL
 {
     [EAGLContext setCurrentContext:self.context];
 
-    Tangram::initialize("scene.yaml");
-    Tangram::loadSceneAsync("scene.yaml");
-    Tangram::setupGL();
+    if (!self.map) {
+        self.map = new Tangram::Map("scene.yaml");
```

Comment:
Updated. Glad I tried manual deletion - it exposed a pretty subtle bug https://github.com/tangrams/tangram-es/pull/810/commits/dacf44e8bdd0d9146363431a2730305ef8d59cbf